core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.spec.ts

```diff
@@ -240,6 +240,31 @@ describe('Customize Interaction Modal Component', () => {
     expect(component.isSaveInteractionButtonEnabled()).toBe(true);
   });
 
+  it('should enable Save Interaction button for End Exploration', () => {
```

Comment:
Mention the scenario in which this action is expected. As in, the save button should be enabled when...

Reply:
added a proper desc for the test